workflow/sanity_workflow.py

```diff
@@ -12,42 +13,56 @@ class SanityWorkflow:
 
     def __init__(html):
         log.info("Inside SanityWorkflow")
+
+        # To get the data from Excel
+        excel_data = ExcelUtils.read_from_excel()
+        testcase_numbers = []
+        for data in excel_data:
+            testcase_numbers.append(data[0])
```

Comment:
excel_data[0] can be used for column number, creation of test case_numbers list is unnecessary

Reply:
Added Work-around to directly use excel_data[0][0],[1][0]...[4][0] as a Test Case Numbers